SCAutolib/controller.py

```diff
@@ -123,6 +134,9 @@ def setup_system(self, install_missing: bool, gdm: bool):
             logger.critical(msg)
             raise exceptions.SCAutolibException(msg)
 
+        if graphical:
```

Comment:
I would suggest to first check if the group is already installed and continue only if it's not yet installed. Most likely we will take care of group installation in downstream test.

Reply:
I don't think that this is a problem. Firstly smart card group is installed in the same way. Also if the group is already installed, the dnf command will still run successfully without failing the test.